tests/optim/test_jacobian.py

```diff
@@ -117,6 +119,26 @@ def forward(self, inputs):
         J = pp.optim.functional.modjac(invnet, input=inputs, **jackwargs)
         assert not pp.hasnan(J)
 
+    def test_lietensor_jacfwd(self):
+        pose = pp.randn_SE3(1).to(device)
+        points = torch.randn(1, 3).to(device)
+        def func(pose, points):
+            return pose @ points
+        out = func(pose, points)
+
+        try: # the torch behavior since ver 2.0.0
+            jac_func = jacfwd(func)
+            jac = jac_func(pose, points)
+            raise AssertionError('should not reach here')
+        except RuntimeError as e:
+            assert 'shapes cannot be multiplied' in str(e)
+
+        with WrappableLT():
+            jac_func = jacrev(func)
+            jac = jac_func(pose, points)
+            assert not pp.hasnan(jac)
```

Comment:
We can assert the jac with LieTensor input is all close to the jac with only Torch.Tensor input if inputs have the same data.

Reply:
Did not find an easy way to do that. It is hard to reproduce the exact computation graph as a LieTensor goes through.